lib/core/DocsLayout.js

```diff
@@ -66,7 +66,7 @@ class DocsLayout extends React.Component {
         metadata={metadata}>
         <div className="docMainWrapper wrapper">
           <DocsSidebar metadata={metadata} />
-          <Container className="mainContainer">
+          <Container className="mainContainer docMainContainer">
```

Comment:
Why not directly add the rules within `docMainContainer` into `mainContainer`? Is there a need for a new class?

Reply:
because the class is used in other page, like blog so it may break the other page if i modify it directly